terraform/modules/quorum-cluster-region/variables.tf

```diff
@@ -98,6 +98,20 @@ DESCRIPTION
   default     = ""
 }
 
+variable "backup_lambda_ssh_private_key_path" {
+  description = <<DESCRIPTION
+SSH private key to be used for authentication.
+DESCRIPTION
+  default     = ""
+}
+
+variable "backup_lambda_ssh_private_key" {
+  description = <<DESCRIPTION
+SSH private key to be used for authentication.
```

Comment:
nit: SSH private key to be used for authentication **by the BackupLambda function**